src/renderRepoCard.js

```diff
@@ -1,9 +1,10 @@
 const { kFormatter, encodeHTML, fallbackColor } = require("../src/utils");
 
 const renderRepoCard = (repo, options = {}) => {
-  const { name, description, primaryLanguage, stargazers, forkCount } = repo;
-  const { title_color, icon_color, text_color, bg_color } = options;
+  const { username, name, description, primaryLanguage, stargazers, forkCount } = repo;
```

Comment:
btw i think its not working because we are destructuring the username from repo variable, but in fetchRepo.js i don't see any "username" returned

![notworkingpr-78](https://user-images.githubusercontent.com/35374649/87663290-36a96080-c781-11ea-84b8-ae06e4569dac.PNG)